src/molecules/account-menu.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './account-menu.module.css';
+
+import { forwardRef } from "react";
+
+const AccountMenu = ({ isLoggedIn, userName, userIcon, onLogout, onLogin }, ref) => {
+  if (isLoggedIn) {
+    return (
+      <div ref={ref} className={`${styles.accountMenu} surface-container on-surface`}>
```

Comment:
```suggestion
      <div ref={ref} className={`${styles.accountMenu} surface-container on-surface-text`}>
```
これでは `background-color` が `on-surface` になってしまいます. `-text` にすると文字色 (`color`) に適用されます. 18 行目も同様にお願いします.